catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -180,7 +180,15 @@ public Metacard getMetacard(String id)
                         TimeUnit.SECONDS.toMillis(10)), false));
         return resultIterable.stream()
                 .collect(Collectors.toMap(result -> result.getMetacard()
-                        .getId(), Function.identity()));
+                        .getId(), Function.identity(), (old, current) -> {
+                    if (LOGGER.isTraceEnabled()) {
```

Comment:
✏️ I don't see a processing hit in the parameter interpolation; I don't know you need to wrap the log in a test.

Reply:
Depending on the toString of the metacard implementation, it could print the entire metacard which could be heavy. Also depending on how many duplicates there are.. it could be called up to n-1 times of the n results.

Reply:
Solid points. 👍